.flake8

```diff
@@ -1,10 +1,16 @@
 [flake8]
-# https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/
-ignore = E203, E266, E501, W503, F403, F401, E402, C901
```

Comment:
these were settings that this particular person used -- not really relevant to kbase